doc/src/notifications.dox

```diff
@@ -110,6 +117,7 @@
  *   - \c hints should contain the following:
  *       - \c category should be "im.received" to categorize the notification to be related to an instant message
  *       - \c urgency should be 1 (normal) since chat messages are not specifically low or high priority
+ *       - \c category should be set to im.received to allow subtle type specific handling
```

Comment:
Did you intent to duplicate this "_category should be set to im.received_" with the similar "_category should be "im.received"_" above?

Same for the next change below.

Reply:
Woah, somehow didn't realize this was already there. Actually added 2013 even. Support has been long time coming :D I'll remove my new lines.